main.py

```
@@ -370,6 +389,44 @@ def _cleanup(done_task: asyncio.Task) -> None:
+                filepath = os.path.join(profile_dir, fname)
+                # 内容相同则跳过写入，保持 mtime 有意义
+                try:
+                    if os.path.isfile(filepath):
+                        with open(filepath, "r", encoding="utf-8") as f:
+                            if f.read().strip() == tag["value"]:
+                                continue
+                except OSError:
+                    pass
+                self.profile_injector.write_tag(filepath, tag["value"])
+        except Exception:
+            pass  # 画像提取失败不阻塞主流程
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)